package.json

```diff
@@ -71,13 +71,16 @@
     "raven-js": "1.1.22",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "sinon": "^2.1.0",
+    "sinon": "^19.0.2",
     "sugar": "1.4.1",
     "ua-parser-js": "0.7.33",
     "uuid-v4.js": "1.0.2",
     "webpack-merge": "^4.2.2",
     "whatwg-fetch": "^2.0.3"
   },
+  "resolutions": {
+    "braces": "3.0.3"
```

Comment:
A last resort, but sadly necessary as `sass` cannot be upgraded any further to resolve the vulnerability in production.